js/src/lib/components/InferenceWidget/shared/WidgetHeader/WidgetHeader.svelte

```diff
@@ -13,7 +13,7 @@
 			<IconLightning classNames="-ml-1 mr-1 text-yellow-500" />
 			Hosted inference API
 		</div>
-		<a target="_blank" href="/docs">
+		<a target="_blank" href="https://api-inference.huggingface.co/">
```

Comment:
note that this doc page is not very good right now, in my opinion (and is probably not very up to date)

When we migrate it to hf.co/docs we should really improve it